multiversx_sdk_cli/cli_contracts.py

```diff
@@ -433,6 +457,18 @@ def query(args: Any):
     utils.dump_out_json(result)
 
 
+def _get_contract_arguments(args: Any) -> Tuple[List[Any], bool]:
```

Comment:
Don't you need to check if `--abi` is provided in this function?

Reply:
indeed. added a check for the abi if `--arguments-file` is provided.